### DISCUSSION TITLE
Cask

### DISCUSSION THREAD

**Discussion by vinaylalam** [Homebrew/discussions#4818] [Casks]
### Output of `brew config`

```shell
brew upgrade git
```

### Output of `brew doctor`

```shell
Error: Cannot install under Rosetta 2 in ARM default prefix (/opt/homebrew)!
To rerun under ARM use:
    arch -arm64 brew install ...
To install under x86_64, install Homebrew into /usr/local.
```

### Description of issue

I am seeing this error in my terminal 

brew upgrade git

Error: Cannot install under Rosetta 2 in ARM default prefix (/opt/homebrew)!
To rerun under ARM use:
    arch -arm64 brew install ...
To install under x86_64, install Homebrew into /usr/local.

### Relevant casks

iDK

**Comment by SMillerDev** [comment]
Have you tried doing the thing it tells you to do?
